testing/describes.js

```diff
@@ -333,6 +338,42 @@ class SandboxFixture {
   }
 }
 
+/** @implements {Fixture} */
+class IntegrationFixture {
+  /** @param {!{body: string}} spec */
```

Comment:
i don't think TODO is needed, especially we recently introduced a new "TODO rule" that we need to create a github issue for each TODO.

this works as is right now. if anyone want to add tests using extension, they will add them wisely.